package/MDAnalysis/analysis/base.py

```diff
@@ -73,10 +75,10 @@ def _setup_frames(self, trajectory, start=None,
         self.stop = stop
         self.step = step
         self.nframes = len(range(start, stop, step))
-
+    
```

Comment:
added whitespace. I'm sure you can teach atom to remove all trailing whitespace on save.